ios/RNSentry.mm

```diff
@@ -195,6 +199,86 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
     resolve(modulesString);
 }
 
+RCT_EXPORT_METHOD(fetchNativePackageName:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    NSString *packageName = [[NSBundle mainBundle] executablePath];
+    resolve(packageName);
+}
+
+- (NSDictionary*) fetchNativeStackFramesBy: (NSArray<NSNumber*>*)instructionsAddr
```

Comment:
Can this be simplified by https://github.com/getsentry/sentry-dart/blob/35005743b3b663c45136b546c2fd11ab2e36fd39/flutter/ios/Classes/SentryFlutterPluginApple.swift#L242 ?

Reply:
It would only simplified the end of the function: https://github.com/getsentry/sentry-react-native/pull/3201/files#diff-ed0b1c4e6c4d33d23050f30ecbe94615487ed47eb7232cedd651092566b9cfa6R259-R265

And also the method `getDebugImages` gets all loaded images which is very inefficient. It retrieves all loaded images but based on the stack addresses only a handful is needed. (We are talking ones/tens vs 100s)